### DISCUSSION TITLE
load trusted root certificates from /usr/share/certs/trusted/

### DISCUSSION THREAD

**Discussion by ip75** [strongswan/strongswan#1325] [General]
There is no way to point in config file `ipsec.conf` path to trusted root certificates installed to OS that will be loaded further while start. The only way to copy files from `/usr/share/certs/trusted/` to `cacerts` directory.

**Comment by strongX509** [comment]
You could make `cacerts` a symbolic link pointing to `/usr/share/certs/trusted`:
````
sudo -s; cd /etc/ipsec.d; rmdir cacerts; ln -s /usr/share/certs/trusted cacerts
````

**Reply by ip75** [reply]
I thought about it but I have my own CA certificates. If I copy them to system CA files location `/usr/share/certs/trusted` after next update OS my CA files will be deleted. I have no way to combine two CA files location. It'd be solution when strongswan scans `cacerts` directory recursively to find CA files. `ln -s /usr/share/certs/trusted cacerts/system_ca`  - something like this. ))

**Reply by strongX509** [reply]
You can import cacerts in the swanctl.conf configuration file via the `authorities` section, see https://docs.strongswan.org/docs/5.9/swanctl/swanctlConf.html#_authorities

In the deprecated ipsec.conf configuration file it is done via a `ca` section, see https://wiki.strongswan.org/projects/strongswan/wiki/CaSection

**Reply by ip75** [reply]
forget about it. I use legacy configuration. I'll try to switch to new one.
